src/main/java/uk/gov/hmcts/ccd/v2/V2.java

```diff
@@ -118,5 +119,8 @@ private Error() {
         public static final String UNAUTHORISED_S2S_SERVICE = "Unauthorised S2S service";
         public static final String CASE_TYPE_INVALID = "Case Type Id is not valid";
         public static final String DATE_STRING_INVALID = "Invalid date sent to search case query ";
+        public static final String PARAM_NOT_NUM = "Parameter is not numeric";
+        public static final String CASE_REFERENCE_NOT_FOUND = "Case Reference Not Found";
```

Comment:
FTAs imply response is `"No case found"` or `"No case found for reference ..."`.

Reply:
Fixed in [54f1d4f](https://github.com/hmcts/ccd-data-store-api/pull/1920/commits/54f1d4fcf3faac512d1a6278a71e82b99881cce4)

Have removed this and returned to standard "No case found"